.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     name: Load dependencies
     strategy:
       matrix:
-        toolchain: [clang16, gcc12]
+        toolchain: [llvm, clang16, gcc12]
```

Comment:
This is just some simplification of the workflow - I had split out the llvm part since occasionally the cache rebuild used to time out when a full rebuild was needed. With the update to enterprise, this is no longer a concern however.